tests/ProxyManagerTest/Autoloader/AutoloaderTest.php

```diff
@@ -5,6 +5,7 @@
 namespace ProxyManagerTest\Autoloader;
 
 use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_MockObject_MockObject;
```

Comment:
PHPUnit 8 removed the BC fallback for `PHPUnit_Framework_MockObject_MockObject` and now only `PHPUnit\Framework\MockObject\MockObject` works (which has been around for a while).

Reply:
Yes, this patch is old, and repository ia still locked onto phpunit 7, since I rely on reflection-based assertions

Reply:
Sorry, for not making this clear: PHPUnit already has `PHPUnit\Framework\MockObject\MockObject` and only provides `PHPUnit_Framework_MockObject_MockObject` for BC.

Reply:
Ah, yup, #441 is on the way